types/postgraphile/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for postgraphile 2.3
```

Comment:
it seems to be v4: 4.0.0-alpha2.30 https://github.com/postgraphql/postgraphql/blob/postgraphile/package.json